metadata-protobuf/proto/WorkingGroups.proto

```diff
@@ -63,8 +63,13 @@ message ModeratePost {
   required string rationale = 2;
 }
 
+message VerifyValidator{
+  required uint64 memberId = 1;
+  required string validatorAccount = 2;
```

Comment:
Hey @chrlschwb the scope has changed:
The verification will come from the membership group instead of HR and the worker will actually be verifying the member profile of the validator accounts not the validator accounts. So there shouldn't be a `validatorAccount` field here.
Also the workers should be able to "unverify" a membership (in case of error). So there should be a:
```suggestion
  required bool is_veryfied = 2;
}
```
instead. As a result the mappings and tests code will have to be changed too.